attractions/chip/checkbox-chip.svelte

```diff
@@ -1,20 +1,64 @@
 <script>
+  /**
+   * @event {{ value: string; checked: boolean; nativeEvent: Event }} change
+   */
   import { createEventDispatcher } from 'svelte';
   import classes from '../utils/classes.js';
 
   let _class = null;
   export { _class as class };
+  /**
+   * A class string to add to the `<input>` element.
+   * @type {string}
+   */
   export let inputClass = null;
+  /**
+   * A class string to add to the underlying Chip element.
+   * @type {string}
+   */
   export let chipClass = null;
 
+  /**
+   * Whether the chip is selected or not.
+   * @type {boolean}
+   */
   export let checked = false;
+  /**
+   * The value of this checkbox. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value) for more information.
+   * Included in events unaltered (won't be stringified if not a string).
```

Comment:
I left that note there just because you wrote it, but now I don't see how it is even valid. In the dispatched event, `value` is taken from `e.target.value`; is that not stringified by the browser?

Reply:
No, it isn't. Perhaps the target is actually the component instance, not the DOM element. But still, bad behaviour!

Reply:
So what do you suggest? Mark the type as `any` or remove the comment?